packages/math-input/src/components/tabbar/icons.tsx

```diff
@@ -8,57 +8,75 @@ type Props = {
 };
 
 const IconAsset = function ({tintColor, type}: Props): React.ReactElement {
-    if (type === "Geometry") {
```

Comment:
Most of these changes are just switching from if/else to switch to be more consistent with `button-assets`